packages/server/src/auth/auth.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { PrismaService } from 'src/prisma.service';
```

Comment:
```suggestion
import { PrismaService } from '../prisma.service';
```
Don't use src in importing since it get compiled into JS in a dist folder